homeassistant/components/bond/light.py

```diff
@@ -113,6 +149,44 @@ async def async_turn_off(self, **kwargs: Any) -> None:
         await self._hub.bond.action(self._device.device_id, Action.turn_light_off())
 
 
+class BondDownLight(BondBaseLight, BondEntity, LightEntity):
+    """Representation of a Bond light."""
+
+    def _apply_state(self, state: dict):
+        self._light = state.get("down_light") and state.get("light")
```

Comment:
`is_on` compared with `self._light == 1`.

Python does indeed made `True` and `False` a subclass of `int` but it is confusing to rely on that in the code.

Reply:
I prefer storing a boolean in `self._light`

Reply:
Since the bond-api is returning `1` or `0` for these values it looks like the integration is using that convention in a few places.  

`light.py`
`        return self._power == 1`
`        return self._light == 1`
`switch.py`
`        return self._power == 1`

I can do a followup PR to convert them to store `True` or `False` in `self._power` and `self._light` instead of `1`/`0`/`None` if you think its worth changing the convention?

Reply:
I have some coverage to improve here. I'll include the result of this conversation in the followup.